tests/lookup/tests.py

```diff
@@ -741,6 +748,13 @@ def test_chain_date_time_lookups(self):
             ordered=False
         )
 
+    def test_time_lookups(self):
+        self.assertQuerysetEqual(
+            Alarm.objects.filter(time__minute=30),
```

Comment:
tests for hour and seconds?
A separate test file like would probably be good  `tests/lookup/test_timefield.py`. Use `setUpTestData()` instead of `setUp()`.